prototype/src/main/webapp/chooseLocation.js

```diff
@@ -28,7 +37,8 @@ function createMarkerForDisplay(lat, lng, content) {
     '<div id="bodyContent">' +
     "<p>Number of Votes: 1</br>" +
     "Voters: You</br></p>" +
-    "<button>VOTE</button><br/><br/>"
+    "<input type=\"checkbox\" id=\"vote\" name=\"vote\" value=\"vote\">" +
+    "<label for=\"vote\"> VOTE</label><br></br>" +
```

Comment:
I would use single quotes here (and also change the rest of the strings here to use single quotes) so you don't have to keep escaping the double quotes. Alternatively, use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), and you won't need the concatenations anymore.

Reply:
Thanks Austin - I'll update it :)